charts/otomi-api/templates/istio/requestauthentication.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: security.istio.io/v1
+kind: RequestAuthentication
+metadata:
+  name: {{ include "otomi-api.fullname" . }}
+  labels:
+{{ include "otomi-api.labels" . | indent 4 }}
+  {{- with .Values.ingress.annotations }}
```

Comment:
Why do we need ingress annotations here?

Reply:
No, these slipped through when copying the template structure. They are removed now.